libs/commons/Common.ml

```diff
@@ -66,6 +66,23 @@ let ( =*= ) = ( = )
  *)
 let ( = ) = String.equal
 
+(*****************************************************************************)
+(* Comparison *)
+(*****************************************************************************)
+
+(* To use `ppx_compare`, it's necessary to have `Ppx_compare_lib.Builtin
```

Comment:
Why using ppx_compare? Why not ppx_deriving and using the 'ord' which seems equivalent.
See https://github.com/ocaml-ppx/ppx_deriving